### DISCUSSION TITLE
Playlist links

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#338] [Show and tell]
I was working through Jenkins CI and made a fork of the iptv-org/iptv project.

https://raw.githack.com/BellezaEmporium/iptv/master/.gh-pages/index.m3u
https://raw.githack.com/BellezaEmporium/iptv/master/.gh-pages/index.category.m3u
https://raw.githack.com/BellezaEmporium/iptv/master/.gh-pages/index.language.m3u
https://raw.githack.com/BellezaEmporium/iptv/master/.gh-pages/index.nsfw.m3u
https://raw.githack.com/BellezaEmporium/iptv/master/.gh-pages/index.region.m3u
https://raw.githack.com/BellezaEmporium/iptv/master/.gh-pages/index.country.m3u

**Comment by goldrake1975** [comment]
buongiorno puoi darmi o far sapere dove trovare lista canali italiani completa?

**Reply by BellezaEmporium** [reply]
https://raw.githack.com/BellezaEmporium/iptv/master/.gh-pages/countries/it.m3u
